dev-util/gdbus-codegen/gdbus-codegen-2.80.4.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+GNOME_ORG_MODULE="glib"
+PYTHON_COMPAT=( python3_{10..12} )
```

Comment:
Maybe this could be updated to 10..13

Reply:
Done in https://github.com/gentoo/gentoo/pull/36834/commits/78e3aa629d8fb95c29ff0603a383e4e123b1dba9